carcara/src/checker/rules/pb_blasting.rs

```diff
@@ -176,8 +176,22 @@ pub fn pbblast_bvugt(RuleArgs { pool, conclusion, .. }: RuleArgs) -> RuleResult
 ///
 /// The expected shape is:
 ///    `(= (bvuge x y) (>= (- (+ sum_x) (+ sum_y)) 0))`
-pub fn pbblast_bvuge(RuleArgs { .. }: RuleArgs) -> RuleResult {
-    Err(CheckerError::Unspecified)
+pub fn pbblast_bvuge(RuleArgs { pool, conclusion, .. }: RuleArgs) -> RuleResult {
+    let ((x, y), ((sum_x, sum_y), constant)) =
+        match_term_err!((= (bvuge x y) (>= (- sum_x sum_y) constant)) = &conclusion[0])?;
+
+    // Get the summation lists
+    let sum_x = get_pbsum(sum_x);
+    let sum_y = get_pbsum(sum_y);
+
+    // Check that the constant is 0
+    let constant: Integer = constant.as_integer_err()?;
+    rassert!(
+        constant == 0,
+        CheckerError::Explanation(format!("Non-zero constant {}", constant))
+    );
```

Comment:
You don't need to check that the constant is 0 manually, `match_term_err!` can do that for you:
```suggestion
    let ((x, y), ((sum_x, sum_y), ())) =
        match_term_err!((= (bvuge x y) (>= (- sum_x sum_y) 0)) = &conclusion[0])?;

    // Get the summation lists
    let sum_x = get_pbsum(sum_x);
    let sum_y = get_pbsum(sum_y);
```